### PR TITLE
[US108 Task151] Change dropdown to select multi-sprint

### PR DESCRIPTION
**Pull request by spark290**
[US108 Task151] Change dropdown to select multi-sprint
screenshot1
screenshot2

### CODE REVIEW

File: react-ui/src/sprint/index.js [hunk 1]
```diff
@@ -23,83 +24,88 @@ const SprintDetail = ({ sprintDetails, attributes, token, projectName }) => {
...
+            <Stack gap={4}>
+                <h3 className="projectName">{projectName}</h3>
+                <div className="d-flex justify-content-center backgroundWhite">
+                    <Select
+                        isMulti
+                        options={options}
+                        className="basic-multi-select"
+                        classNamePrefix="select"
+                        onChange={handleSelect}
+                        placeholder="Select Metrics"
```
Comment (AdvaitChirmule) [inline react-ui/src/sprint/index.js]: Could you please change the placeholder to "Select Sprints" instead?

Comment (spark290) [reply react-ui/src/sprint/index.js]: Modified

**Review by AdvaitChirmule — APPROVED** [review]
Looks good to me

**Review by shahih25 — APPROVED** [review]
lgtm

**Pull request merged**